pkg/lib/common.ts

```diff
@@ -1,3 +1,5 @@
+export type AnyStr = string | Uint8Array;
```

Comment:
Hmm, `StrOrBytes`?

Reply:
I borrowed the name from Python.

Reply:
Hm, is that still from a time when `str` was really a `bytes[]`? In Python 3 this isn't the case, and thus feels anachronistic. Definitively not the case in JS, UInt8Array is not a string.

Anyway, not important other than exporting the symbol as public API (but see discussion below).

Reply:
https://docs.python.org/3/library/typing.html#typing.AnyStr

Interesting!! This is not what I thought it was!  It is indeed just `str | bytes` effectively, but it enforces that all instances where it appears in a single declaration are the same.  The `AnyStr` I've added here is not the same thing at all.

I'll name it StrOrBytes.

Reply:
Done.